commit 68d744ae40a898b8e6c5771caae5fe726122437a (refs/changes/39/4739/1)
Author: Jared Morrow <jm@joyent.com>
Date:   2018-08-23T15:23:31-06:00 (1 year, 2 months ago)
    
    MANTA-3877 Add no_reindex option to manta bucket creation for GC trigger update

diff --git a/lib/server.js b/lib/server.js
index c019a11..18deb9a 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -573,7 +573,7 @@ function updateBucket(options) {
         }
 
         options.clients.array.forEach(function (client) {
-            client.updateBucket(name, cfg, function (err2) {
+            client.updateBucket(name, cfg, opts, function (err2) {
                 log.debug({
                     err: err2
                 }, 'updateBucket: returned');
@@ -647,7 +647,7 @@ function delBucket(options) {
         }
 
         options.clients.array.forEach(function (client) {
-            client.delBucket(name, function (err2) {
+            client.delBucket(name, opts, function (err2) {
                 log.debug({
                     err: err2
                 }, 'delBucket: returned');
